app/controllers/calls_controller.rb

```diff
@@ -0,0 +1,25 @@
+class CallsController < ApplicationController
+  skip_before_action :verify_authenticity_token
+  #skip_before_filter :authenticate_user!, :only => "reply"	
+  
+  def begin
+  	boot_twilio
+  	beginCall = BeginCall()
+  	response = VoiceResponse(beginCall)
```

Comment:
When initializing objects, you need to use .new after the class name.
[Prefer snake case over camel case for variables](https://github.com/Invoca/style-guide/tree/master/ruby#use-snake_case-for-methods-and-variables)
`VoiceResponse.new(begin_call)`